css/theme.less

```diff
@@ -49,6 +49,11 @@ body:before {
     background: @ini_background;
 }
 
+/* Content */
+.site-content{
+    z-index: 0;
```

Comment:
Minor: The commit message has a typo and mentions "z-height".

Reply:
Not sure if I can change a commit message though?